examples/tpm_test.h

```diff
@@ -103,7 +103,8 @@ static const char pemFileKey[] = "key.pem";
         extern double current_time(int reset);
         return current_time(reset);
     #elif defined(_WIN32)
-        return ((double)GetTickCount64())/1000.0;
+        (void)reset;
+        return (0.0 + GetTickCount64())/1000.0;
```

Comment:
What was the original error/warning? I don't like the 0.0 workaround. Can you try using the following? `return ((double)GetTickCount64())/1000.0F;`

Reply:
it doesn't like that we're casting the return value of a function, the error was -Werror=bad-function-cast, let me see if it complains if I put it in a variable first

Reply:
yeah compiler is fine with it as long as it's stored in a variable fist, should be good now